lang/de.json

```diff
@@ -405,33 +444,44 @@
     "Create User": "Benutzer anlegen",
     "created": "hat erstellt",
     "Created At": "Erstellt am",
+    "Created By": "Erstellt von",
     "Credit Limit": "Kreditlimit",
+    "Credit Line": "Kreditlinie",
     "Credit line": "Kreditlinie",
     "Creditor Identifier": "Gläubiger-Identifikationsnummer",
+    "Creditor Number": "Gläubigernummer",
     "Creditor number": "Kreditor-Nummer",
```

Comment:
decide whether it should be translated to "Gläubigernummer" or "Kreditor-Nummer"